Sources/MapboxMaps/Style/Style.swift

```diff
@@ -167,25 +167,25 @@ public final class Style: StyleProtocol {
     }
 
     /**
-     Retrieves a source from the map
+     Retrieves a `source` from the map
      - Parameter id: The id of the source to retrieve
      - Parameter type: The type of the source
 
      - Returns: The fully formed `source` object of type equal to `type`.
-     - Throws: StyleError or type conversion errors
+     - Throws: `StyleError` or type conversion errors
      */
     public func source<T>(withId id: String, type: T.Type) throws -> T where T: Source {
         let sourceProps = try sourceProperties(for: id)
         return try type.init(jsonObject: sourceProps)
     }
 
     /**
-     Retrieves a source from the map
+     Retrieves a `source` from the map
 
      This function is useful if you do not know the concrete type of the source
      you are fetching, or don't need to know for your situation.
 
-     - Parameter id: The id of the source to retrieve.
+     - Parameter id: The id of the `source` to retrieve.
      - Returns: The fully formed `source` object.
      - Throws: Type conversion errors.
```

Comment:
```suggestion
     - Throws: ``StyleError`` if there is a problem getting the source data.
     - Throws: ``TypeConversionError`` if there is a problem decoding the source of given `id`.
```
IMO this should also be updated while we are at it for consistency.